pyxlma/lmalib/flash/properties.py

```diff
@@ -8,9 +8,9 @@
 def local_cartesian(lon, lat, alt, lonctr, latctr, altctr):
     Re = 6378.137e3           #Earth's radius in m
     latavg, lonavg, altavg = latctr, lonctr, altctr
-    x = Re * (np.radians(lonavg) - np.radians(lon)) * np.cos(np.radians(latavg))
-    y = Re * (np.radians(latavg) - np.radians(lat))
-    z = altavg - alt
+    x = Re * (np.radians(lon) - np.radians(lonavg)) * np.cos(np.radians(latavg))
+    y = Re * (np.radians(lat) - np.radians(latavg))
+    z = alt - altavg
```

Comment:
Yikes, good catch on the order of subtraction. Did you verify the result of this is oriented the same as a lon-lat-alt plot of the points? It seems like it should be, but wanted to be sure.

Reply:
I did, both the local cartesian and original coordinates show the same orientation of the data now.